cmd/snap/cmd_buy.go

```diff
@@ -94,10 +94,12 @@ func buySnap(snapName string) error {
 		if e, ok := err.(*client.Error); ok {
 			switch e.Kind {
 			case client.ErrorKindNoPaymentMethods:
-				return fmt.Errorf(i18n.G(`You do not have a payment method associated with your account, visit https://my.ubuntu.com/payment/edit to add one.
-Once completed, return here and run 'snap buy %s' again.`), snap.Name)
+				return fmt.Errorf(i18n.G(`You need to have a payment method associated with your account in order to buy a snap, please visit https://my.ubuntu.com/payment/edit to add one.
+
+Once you’ve added your payment details, you just need to run 'snap buy %s' again.`), snap.Name)
 			case client.ErrorKindTermsNotAccepted:
-				return fmt.Errorf(i18n.G(`Please visit https://my.ubuntu.com/payment/edit to agree to the latest terms and conditions.
+				return fmt.Errorf(i18n.G(`In order to buy oracledb, you need to agree to the latest terms and conditions. Please visit https://my.ubuntu.com/payment/edit to do this.
```

Comment:
Err, we should not use `oracledb` literally here, right? We need to format the argument we got.